app/views/layouts/application.html.erb

```diff
@@ -105,7 +107,9 @@
 
       AJAX_LOADING: _('Loading ...'),
       AJAX_UNABLE_TO_LOAD_TEMPLATE_SECTION: _('Unable to load the section\'s content at this time.'),
-      AJAX_UNABLE_TO_LOAD_TEMPLATE_SECTION_QUESTION: _('Unable to load the question\'s content at this time.')
+      AJAX_UNABLE_TO_LOAD_TEMPLATE_SECTION_QUESTION: _('Unable to load the question\'s content at this time.'),
+
+      OTHER_ORG_ID: other_org.present? ? other_org.id : 0
```

Comment:
Does this need to be a constant for the JS or could it just be included in the page for the views it's needed on?

Reply:
I could have, but it seemed reasonable to load the variable only once. The shared `my_org.js` is referenced from several different pages. Each of those pages would need to have that variable stored in a hidden input field. We also have situations where the my_org control is loaded multiple times on one render which could cause collisions/discrepancies if there were multiple sources for the value.

Reply:
As far as I'm aware, the typeahead orgs is this partial (e.g. https://github.com/DMPRoadmap/roadmap/blob/master/app/views/shared/_my_org.html.erb) which gets rendered in four views. Why don't we create a hidden input for when allow_other_orgs is true to be set to the other org id instead of mixing concerns between Ruby and JS? Going for this approach, will reduce the need of an extra query to determine the other org id when the page rendered does not really need it. Does it make sense?